packages/suite/src/views/dashboard/components/AssetsView/components/AssetCard.tsx

```diff
@@ -124,7 +126,7 @@ export const AssetCard = ({
     const theme = useTheme();
     const { setCoinFilter, setSearchString } = useAccountSearch();
 
-    const handleLogoClick = () => {
+    const handleGoToCoinDetailClick = () => {
```

Comment:
nitpick: There is `handleRowClick` in `AssetRow` and so it would make sense to have `handleCardClick` here

Reply:
:heavy_check_mark: